scripts/blueprints/dockerhub/node.md

```diff
@@ -0,0 +1,148 @@
+EXPOSE 8888
+```
+
+You can then build and run the Docker image:
+
+```console
+$ docker build -t my-nodejs-app .
+$ docker run -it --rm --name my-running-app my-nodejs-app
+```
+
+If you prefer Docker Compose:
+
+```yml
+version: "2"
+services:
+  node:
+    image: "balenalib/raspberrypi3-node:10"
+    user: "node"
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=production
+    volumes:
+      - ./:/home/node/app
+    expose:
+      - "8081"
+    command: "npm start"
+```
+
+You can then run using Docker Compose:
+
+```console
+$ docker-compose up -d
+```
+
+Docker Compose example copies your current directory (including node_modules) to the container.
+It assumes that your application has a file named [`package.json`](https://docs.npmjs.com/files/package.json)
+defining [start script](https://docs.npmjs.com/misc/scripts#default-values).
+
+### Run a single Node.js script
+
+For many simple, single file projects, you may find it inconvenient to write a
+complete `Dockerfile`. In such cases, you can run a Node.js script by using the
+Node.js Docker image directly:
+
+```console
+$ docker run -it --rm --name my-running-script -v "$PWD":/usr/src/app -w /usr/src/app balenalib/raspberrypi3-node:10 node your-daemon-or-script.js
+```
+
+### Verbosity
+
+Prior to 8.7.0 and 6.11.4 the docker images overrode the default npm log
+level from `warn` to `info`. However due to improvements to npm and new Docker
+patterns (e.g. multi-stage builds) the working group reached a [consensus](https://github.com/nodejs/docker-node/issues/528)
+to revert the log level to npm defaults. If you need more verbose output, please
+use one of the following methods to change the verbosity level.
+
+#### Dockerfile
+
+If you create your own `Dockerfile` which inherits from the `node` image you can
+simply use `ENV` to override `NPM_CONFIG_LOGLEVEL`.
+
+```dockerfile
+FROM balenalib/raspberrypi3-node:10
+ENV NPM_CONFIG_LOGLEVEL info
+...
+```
+
+#### Docker Run
+
+If you run the node image using `docker run` you can use the `-e` flag to
+override `NPM_CONFIG_LOGLEVEL`.
+
+```console
+$ docker run -e NPM_CONFIG_LOGLEVEL=info balenalib/raspberrypi3-node:10 ...
+```
+
+#### NPM run
+
+If you are running npm commands you can use `--loglevel` to control the
+verbosity of the output.
+
+```console
+$ docker run balenalib/raspberrypi3-node:10 npm --loglevel=warn ...
+```
+
+# Image Variants
+
+The `node` images come in many flavors, each designed for a specific use case.
+
+## `node:<version>` or `node:<version>-run`
+
+This is the defacto image. The `run` variant is designed to be a slim and minimal variant with only runtime essentials packaged into it.
+
+## `node:<version>-build`
+
+The build variant is a heavier image that includes many of the tools required for building from source. This reduces the number of packages that images that derive from it need to install, thus reducing the overall size of all images on your system.
```

Comment:
This section of the sentence doesn't make sense, consider revising. "...number of packages that images that derive from it need to install...."

Reply:
I've rephrased this section, is that ok?